From b1381949222f4bd0fac599ad22a37f8131b08a67 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 10 Jun 2022 16:15:29 +0000
Subject: [PATCH 445/462] Mantis 0062866 extension: ID1 dont dispense after
 reading ID cards

DeskoID1.exe
V1.0.0.7
- use SDK V1.10.1; with FW: 0C0C0101.00010029
  - \\file07\Abteilungen\MSSD\Airlines\massai\Projekte\Platform\Build\Globals\GlobalsLib\desko\DeviceApi\1.10.1
  - desko_usb.dll has changed
- use DDA_DOS_ISDOCEJECTED for dispenser status


git-svn-id: svn://localhost/SelfServiceBuild/trunk@861 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index 835cea5..55a581f 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -553,6 +553,10 @@ set DIR_EPASS102014=%DIR_GLOBALSLIB%\desko\EPassApi2\102.0.14
 set INC_EPASS102014=%DIR_EPASS102014%\include
 set LIB_EPASS102014=%DIR_EPASS102014%\lib\win32\ePassApi2.lib
 
+set DIR_DESKODEVICEAPI1101=%DIR_GLOBALSLIB%\desko\DeviceApi\1.10.1
+set INC_DESKODEVICEAPI1101=%DIR_DESKODEVICEAPI1101%\include
+set LIB_DESKODEVICEAPI1101=%DIR_DESKODEVICEAPI1101%\lib\Win32\DDevApi.lib
+
 rem ######################################################################
 rem # Externe Bibliotheken
 rem ######################################################################
-- 
2.41.0.windows.1

